verifier/Cargo.toml

```diff
@@ -17,12 +17,14 @@ rust-version = { workspace = true }
 alloc = ["pem-rfc7468/alloc", "dep:const-oid", "dep:p256", "dep:x509-cert", "tcb"]
 tcb = ["dep:p256", "dep:serde_json", "dep:serde", "dep:der", "dep:hex", "advisories"]
 advisories = ["dep:serde_json"]
+x509 = ["dep:mbedtls"]
```

Comment:
I thought about using a separate crate, but that means another crate to publish. And with an eventual long term desire to replace mbedtls it seemed like a feature would be easier to isolate and alter as needed.